payu/experiment.py

```diff
@@ -596,6 +638,17 @@ def run(self, *user_flags):
 
                 shutil.copyfile(src, dest)
 
+            for fname in (self.job_fname, self.env_fname):
+
+                src = os.path.join(self.control_path, fname)
+
+                # Assumes .yaml extension
+                dest = os.path.join(error_log_dir,
+                                    fname[:-5] + '.' + job_id + fname[-5:])
+                print(src, dest)
```

Comment:
Testing diagnostic?  Delete?

Reply:
I just copied your code. You have this for the `.err` and `.out`. I have coalesced them into a single code block now, but happy to delete the `print` statement if you don't think it is still necessary.

Reply:
Haha I see... I think that print statement is weird, but I think it was to give a bit of feedback on the sweep?

I guess leave for now, if it's weird we can take it out.